docs/book/user-guide/starter-guide/using-project-templates.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Rocketstart your ZenML journey!
+---
+
+# ZenML project templates
+
+What would you need to get a quick understanding of the ZenML framework and start building your ML pipelines? The answer is one of ZenML project templates to cover major use cases of ZenML: a collection of steps and pipelines and, to top it all off, a simple but useful CLI. This is exactly what the ZenML templates are all about!
+
+## List of ZenML Project Templates
+
+| Project Template [Short name] | Tags | Description |
```

Comment:
On Gitbook, you can change the width of this table to full-width in order to make it a bit more readable if you want to.

Reply:
Did so 👍🏼